python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -104,5 +112,8 @@ def _build_treelite_classifier(m, data, arg={}):
 
 
 def _treelite_fil_accuracy_score(y_true, y_pred):
-    y_pred_binary = y_pred > 0.5
-    return cuml.metrics.accuracy_score(y_true, y_pred_binary)
+    if isinstance(y_pred, np.ndarray):
```

Comment:
Is special handling really needed here?

Reply:
Yes, we'll need this to convert `DeviceNDArray` into `np.ndarray` as the accuracy function cannot take gpuarray as input (will have `AttributeError: 'DeviceNDArray' object has no attribute 'astype'` error at `File "cuml/metrics/accuracy.pyx", line 62, in cuml.metrics.accuracy.accuracy_score
    ground_truth = ground_truth.astype(np.int32)')